docs/reference/transform/transforms-at-scale.asciidoc

```diff
@@ -0,0 +1,197 @@
+[role="xpack"]
+[[transform-scale]]
+= Working with {transforms} at scale
+++++
+<titleabbrev>{transforms-cap} at scale</titleabbrev>
+++++
+
+{transforms-cap} convert existing {es} indices into summarized indices, which 
+provide opportunities for new insights and analytics. The search and index 
+operations performed by {transforms} use standard Elasticsearch features so 
+similar considerations for working with Elasticsearch at scale are often 
+applicable to transforms. If you experience performance issues, start by 
+identifying the bottleneck areas (search, indexing, processing, or storage) then 
+review the relevant considerations in this guide to improve performance. It also 
+helps to understand how {transforms} work as different considerations apply 
+depending on whether or not your transform is running in continuous mode or in 
+batch.
+
+In this guide, you’ll learn how to:
+
+* Understand the impact of configuration options on the performance of 
+  {transforms}.
+
+**Prerequisites:**
+
+These guildelines assume you’re already familiar with: 
+
+* <<transform-overview,How to create {transforms}>>.
+* <<transform-setup,How to set up {transforms}>>.
+* <<transform-checkpoints,How {transform} checkpoints work in continuous mode>>.
+
+The following considerations are not sequential – the numbers help to navigate 
+between the list items; you can take action on one or more of them in any order. 
+Most of the recommendations apply to both continuous and batch {transforms}. If 
+a list item only applies to one {transform} type, this exception is highlighted 
+in the description.
+
+The keywords in parenthesis at the end of each recommendation title indicates 
+the factor that may be improved by following the given recommendation.
```

Comment:
> the factor that may be improved by following the given recommendation.

I found myself wondering what a "factor" was.  Is it equivalent to the "bottleneck areas" mentioned earlier? If so, it might be helpful to clarify that here so readers don't get derailed wondering what it means.